mesheryctl/main.go

```diff
@@ -22,7 +22,8 @@ var version = "dev"
 var commitsha = "SHA"
 
 func main() {
-	cmd.Build = version
+	cmd.BuildClient = version
+	cmd.BuildServer = version
```

Comment:
It looks like you're setting the server version to always be the same of the client. The point of this feature is to acknowledge that in a given Meshery deployment, the two of these may differ in version number. `mesheryctl` needs to have a way of retrieving the server version from Meshery itself.

Reply:
@anandrkskd does this make sense?

Reply:
Yes sir, Got it, But I had conversation with @girishranganathan about it on the development meeting, and he asked me to just add the server version same as client for now and when we will develop the api for meshery server then we will replace it with actual server version and for now we are also publishing meshery and mesheryctl binary together so both of them carry the same version.
But I guess, for now we can add a new serververion tag in goreleaser for server version in versionCmd.